ros/processor/insights_engine_result_consumer.py

```diff
@@ -159,7 +159,8 @@ def process_report(self, host, reports, utilization_info, performance_record):
                         'max_io': 0,
                         'io': {}
                     }
-
+                del performance_record['instance_type']
+                del performance_record['region']
                 get_or_create(
```

Comment:
@r14chandra Could you add a comment above, stating the reason for the `del` statements?